_pages/coc.md

```diff
@@ -74,7 +80,14 @@ communications pertaining to community business.
 
 ## Contact Us ##
 
-We won't share your contact information or the contents of your message with anyone other than conference executives.
+You have several options for contacting us:
+
+- send a private Slack message to Evy Kassirer (trained incident responder) or any other conference organizer
+- direct message [Evy](https://twitter.com/EvyKassirer) on Twitter
```

Comment:
See my comment in https://git.uwaterloo.ca/starcon/exec-eyes-only/issues/148

Reply:
thanks! replied there